README.md

```diff
@@ -4,9 +4,10 @@
     <br />
     <a href="https://pypi.org/project/manim/"><img src="https://img.shields.io/pypi/v/manim.svg?style=flat" alt="PyPI Latest Release"></a>
     <a href="http://choosealicense.com/licenses/mit/"><img src="https://img.shields.io/badge/license-MIT-red.svg?style=flat" alt="MIT License"></a>
```

Comment:
```suggestion
    <a href="http://choosealicense.com/licenses/mit/"><img src="https://img.shields.io/badge/license-MIT-red.svg?style=flat&logo=pypi" alt="MIT License"></a>
```
this also ![](https://img.shields.io/badge/license-MIT-red.svg?style=flat&logo=pypi)